integration-tests/code-generator/src/main/java/com/squareup/anvil/test/TestCodeGenerator.kt

```diff
@@ -37,13 +37,47 @@ class TestCodeGenerator : CodeGenerator {
           class GeneratedClass 
         """.trimIndent()
 
+        @Language("kotlin")
+        val contributedInterface = """
+          package $generatedPackage
+          
+          import com.squareup.anvil.annotations.ContributesTo
+
+          @ContributesTo(Unit::class)
+          interface ContributedInterface 
+        """.trimIndent()
+
+        @Language("kotlin")
+        val contributedModule = """
+          package $generatedPackage
+          
+          import com.squareup.anvil.annotations.ContributesTo
+          import dagger.Module 
+
+          @ContributesTo(Unit::class)
+          @Module
+          interface ContributedModule 
```

Comment:
Without the fix in the first commit, this generated code would cause failures in the compiler plugin.